src/migrations/00029-add-fk-indexes.sql

```diff
@@ -0,0 +1,15 @@
+CREATE INDEX CONCURRENTLY IF NOT EXISTS "application_release_idx"
+ON "application" ("should be running-release");
+CREATE INDEX CONCURRENTLY IF NOT EXISTS "application_application_type_idx"
+ON "application" ("application type");
+
+CREATE INDEX CONCURRENTLY IF NOT EXISTS "device_is_running_release_idx"
+ON "device" ("is running-release");
+CREATE INDEX CONCURRENTLY IF NOT EXISTS "device_should_be_running_release_idx"
+ON "device" ("should be running-release");
+
+CREATE INDEX CONCURRENTLY IF NOT EXISTS "gateway_download_device_idx"
+ON "gateway download" ("is downloaded by-device");
+
+CREATE INDEX CONCURRENTLY IF NOT EXISTS "release_tag_release_idx"
+ON "release tag" ("release");
```

Comment:
I guess that this provides a measurable speed improvement on top of the composite release-tag_key unique index?

Reply:
Nope, this is a mistake - fixed it now